packages/core/src/components/toast/toast.md

```diff
@@ -17,7 +17,7 @@ Similarly, focusing the toast (for example, by hitting the `tab` key) halts the
 
 You can add one additional action button to a toast. You might use this to provide an undo button, for example.
 
-You can also apply the same visual intent styles to `Toast`s that you can to [`Button`s](#core/components/button.css).
+You can also apply the same visual intent styles to `Toast`s that you can to [`Buttons-`](#core/components/button.css).
```

Comment:
the original code was correct, albeit a little awkward. we use this kind of syntax in many places in the docs, where the name of the component is emphasized with code backticks. `Buttons` doesn't make sense in this style because that is not a component (there are some pluralized components, though, like `Tabs`).

also there is a stray `-` here